webknossos/webknossos/dataset/properties.py

```diff
@@ -113,6 +113,9 @@ class LayerViewConfiguration:
     is_in_edit_mode: Optional[bool] = None
     """Enable the histogram edit mode. The WEBKNOSSOS default is False."""
 
+    mapping: Optional[Dict[str, str]] = None
+    """Enables ID mapping for a segmentation layer and applies the selected mapping by default. The default WK behavior is to disable ID mapping. Expected values is a Dict with {"name": my_mapping_name, "type": "hdf5"}."""
```

Comment:
@MichaelBuessemeyer do you know if the type needs to be written in capital letters here, HDF5?

Reply:
Good catch. It needs to be in caps. See
https://github.com/scalableminds/webknossos/blob/c50a81860b5e9882b24677e5a750b10f63c5d6a8/frontend/javascripts/oxalis/store.ts#L396

Reply:
```suggestion
    """Enables ID mapping for a segmentation layer and applies the selected mapping by default. The default WK behavior is to disable ID mapping. Expected values is a Dict with {"name": my_mapping_name, "type": "HDF5"}."""
```